src/backend/base/langflow/components/vectorstores/astradb_graph.py

```diff
@@ -248,10 +256,17 @@ def _add_documents_to_vector_store(self, vector_store) -> None:
             logger.debug("No documents to add to the Vector Store.")
 
     def _map_search_type(self) -> str:
+        if self.search_type == "Similarity":
+            return "similarity"
         if self.search_type == "Similarity with score threshold":
             return "similarity_score_threshold"
         if self.search_type == "MMR (Max Marginal Relevance)":
             return "mmr"
+            return "similarity_score_threshold"
```

Comment:
There are two returns here.

Reply:
Oi Gabriel - certainly, There is also a regression issue that pre-dates me. Looks like the class name on the __init.py__ differs from the astradbvectorgraphcomponent. Should I change that as well so the unit tests pass?